### PR TITLE
Bc4wp cron sync to staging

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-3641](https://jira.bigcommerce.com/browse/DEVDOCS-3641)

## Created new endpoint
* Added article drafted up by MT and converted to Markdown and applied style guide.

### CODE REVIEW

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The following are different methods that can be used for adding a new cron job to the server.
+
+**Cron job manager**
+
+There are plenty of job managers available for hosting providers like [HostGator](https://www.hostgator.com/) or [Cloudways](https://www.cloudways.com/en/).
+
+>**Note:** We recommend that you contact the hosting’s tech support for proper setup. Each hosting provider has specific detailed knowledge bases that cover how to set up cron jobs.
+
+The following example is for HostGator but it also can be used for any other hosting with CPanel: 
+1. Login to CPanel.
```
Comment (markcmurphy) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:40]:
```suggestion
1. Log in to cPanel.
```
- cPanel is the official spelling of the product: https://cpanel.net/
- "Log in" is the verb form. Google style guide prefers "sign in" over "log in," but that is totally optional. "login (noun or adjective), log in (verb). For the verb form, _sign in_ is generally better, but if you're documenting a tool that uses the term _log in_, then use the term that the tool uses."

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for WordPress</h3></div>
+
+# BigCommerce for WordPress Cron
+
+This article provides a general description for WP-Cron and how to change its behavior in order to make it more reliable.
```
Comment (markcmurphy) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:5]:
```suggestion
This article provides a general description of WP-Cron and how to change its behavior to make it more reliable.
```
Where possible, don't use _in order to_; instead, use _to_. Occasionally, _in order to_ clarifies meaning or makes something easier to read.

**Review by traci707 — COMMENTED** [review]
I did an editorial review only. I didn't test it.

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for WordPress</h3></div>
+
+# BigCommerce for WordPress Cron
+
+This document provides a general description for WP-Cron and how to change its behavior to improve its reliability.
+
+BigCommerce for WordPress plugin has update/sync logic which is based on cron jobs (WP-Cron). WP-Cron is built-in logic for processing postponed and background tasks. While that logic is great and flexible it also has some problems which can affect import product logic.
```
Comment (bc-andreadao) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:7]: Is it accurate to say here that server cron is the BigCommerce for WordPress plugin?

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## WP-Cron versus Server Cron
+WordPress has its own cron system for scheduling tasks such as checking for updates, scheduling a post for publication, and deleting comments from the trash. All cron jobs are handled by WP-Cron.
+
+The logic has the name WP-Cron, but it's not a real server cron job. The difference is:
```
Comment (bc-andreadao) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:12]:
```suggestion
Differences between server cron and WP-Cron include:
```

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- Server cron runs automatically and is fired by server processes. There's no need for an external source for events firing
+- WP-Cron only runs when someone is on the site
+
+<p>Here's an example:</p>
+
+You schedule a job to run once an hour with WP-cron and WordPress will set an execution time for that job. Yet, the job won't be performed until there's site-traffic or there's an external request to WordPress. 
+
+Because of this, WP-Cron jobs can be postponed or not performed at all. So if you've set a job with the usual server cron it will be performed even if there isn't site traffic or external requests.
+
+## WP-Cron enhance with Server Cron
```
Comment (bc-andreadao) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:22]:
```suggestion
## Using Server Cron to enhance WP-Cron
```

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+BigCommerce for WordPress plugin has update/sync logic which is based on cron jobs (WP-Cron). WP-Cron is built-in logic for processing postponed and background tasks. While that logic is great and flexible it also has some problems which can affect import product logic.
+
+## WP-Cron versus Server Cron
+WordPress has its own cron system for scheduling tasks such as checking for updates, scheduling a post for publication, and deleting comments from the trash. All cron jobs are handled by WP-Cron.
+
+The logic has the name WP-Cron, but it's not a real server cron job. The difference is:
+- Server cron runs automatically and is fired by server processes. There's no need for an external source for events firing
+- WP-Cron only runs when someone is on the site
+
+<p>Here's an example:</p>
```
Comment (bc-andreadao) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:16]: Flip main idea or focus of paragraph to our Server Cron?

File: docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md (hunk 3 above)
Comment (bc-tgomez) [reply docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md:7]: It's not a plugin but a server cron job. I'll check with Paul from MT to verify. @bc-andreadao Thanks for all the great feedback!

**Comment by slsriehl** [comment]
approved!

**Pull request merged**